samples/client/petstore/typescript-angular-v4.3/npm/model/order.ts

```diff
@@ -33,9 +33,7 @@ export interface Order {
 
 }
 export namespace Order {
-    export enum StatusEnum {
-        Placed = <any> 'placed',
-        Approved = <any> 'approved',
-        Delivered = <any> 'delivered'
-    }
+    export type StatusEnum = 'placed' | 'approved' | 'delivered';
```

Comment:
Do you know why you got these changes?

Reply:
I guess because the v4.3 was not re-generated after the changes introduced in https://github.com/swagger-api/swagger-codegen/pull/6233
this means that 4.3 somehow is not checked by CI. @wing328 can you fix that?